examples/cluster/plot_digits_linkage.py

```diff
@@ -22,6 +22,10 @@
 # Authors: Gael Varoquaux
 # License: BSD 3 clause (C) INRIA 2014
 
+# We utilize the digits dataset which has 1800 entries.
+# We will only use 800 entries for the sake of speeding
+# up the calculations but feel free to use the whole dataset.
+
```

Comment:
If we avoid sampling the corpus and remove the function `nudge_images`, we have to delete this comment.